### DISCUSSION TITLE
Discussions Private Categories and/or Permissions

### DISCUSSION THREAD

**Discussion by eliasmbd** [community/community#76517] [Discussions] [answered]
Question

My colleagues and I are currently looking at options to migrate our community. Despite being an open source community we sometimes find value in private conversations between members and especially for us staff that maintain the project.

Does Discussions allow us to do any of the following and if not what are some of the workarounds you might have implemented: 

1. Create private categories for staff and internal comms to coordinate responses to FAQ for example.
2. Assign user/user group permissions for different categories based on labels or other features
3. A chat feature similar to slack integrated to discussions

I appreciate your insights.

**Comment by Kolass2004** [comment, marked as answer]
Discussions is a powerful platform for fostering community engagement, but it does have limitations when it comes to private conversations and advanced permissions management. Here's how you might work around these limitations:

1. Private Categories for Staff:Discussions doesn't natively support private categories. However, you can create a workaround by utilizing tags or labels to mark certain topics as "internal" or "staff-only." You can then communicate internally within those threads while keeping them hidden from the public eye. Moderators or staff members can enforce this by regularly checking and managing the tags.

2. User Permissions for Categories: Discussions doesn't offer granular user permissions for categories. However, you can utilize roles and permissions creatively. For instance, you might have a "Staff" role that grants access to specific internal discussions. You can manually assign this role to trusted members or integrate with a third-party service that provides more granular permissions control.

3. Integrated Chat Feature:While Discussions doesn't have a built-in chat feature like Slack, you can integrate third-party chat solutions such as Slack, Discord, or even a self-hosted chat application. You can then create channels or rooms for real-time communication among staff members. Integrating these tools may require some technical setup, but it can provide the chat functionality you're looking for alongside Discussions.

Remember, while these workarounds can help achieve some level of privacy and internal communication within Discussions, they may not offer the same seamless experience as dedicated tools designed for private conversations and advanced permissions management. Assess your community's needs and consider if integrating external tools or customizing Discussions meets those needs effectively.

**Reply by eliasmbd** [reply]
@Kolass2004 Thank you for getting back to us on this. This definitely with help us make our decisions.
